src/config/dynamic-theme-fixes.config

```diff
@@ -11792,6 +11792,41 @@ INVERT
 
 ================================
 
+manifold.markets
+
+CSS
```

Comment:
These aren't good IMO, fixes shouldn't use the original value as a fix, they still should be modified. You could try to transform these fixes into `property: ${rgb(...)} !important;`, as that's generally how a fix looks like.